components/social-follow/social-follow.snip.html

```diff
@@ -18,6 +18,9 @@
 
 
 {{#social-follow}}
+{{#header}}
```

Comment:
You are correct , The reason i wanted this to stay in the template (the respective page) is to easily add a heading level (You might also need a wrapping div to keep the component markup in a division). I think this should be a heading. You can check for other files on how we passing heading level via a config. 
i think default level could be h3